wimm.Secundatives/Extensions/FlattenExtensions.cs

```diff
@@ -17,6 +17,8 @@ public static class FlattenExtensions
         /// The un-nested result of inspecting the outer result for error and returning it if it exists or the nested result itself otherwise 
         /// </returns>
         public static Result<TValue, TErr> Flatten<TValue, TErr>(this Result<Result<TValue, TErr>, TErr> result)
+            where TValue : notnull
```

Comment:
Yea I think this is breaking

Reply:
All good.

> The notnull constraint is available starting in C# 8.0 for code compiled in a nullable enable context. Unlike other constraints, if a type argument violates the notnull constraint, the compiler generates a warning instead of an error. Warnings are only generated in a nullable enable context.
> \- https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/where-generic-type-constraint